### DISCUSSION TITLE
Rotated hevc videos?

### DISCUSSION THREAD

**Discussion by danergo** [mifi/lossless-cut#1740] [Q&A]
I'm creating hevc videos with DJI Action4 (mounted upside down).

Now unfortunately my lg TV is playing it back exactly upside down oriented.

I was happy to see the orientation option in this app, however it doesn't change anything - so to say I was happy too early:

Tried with:
- no change rotation: lgtv plays upside down
- 180degree rotation: same
- 0degree rotation: same

It seems lgtv is not respecting this option. 

Can you verify this is a metadata option change really? And not some pixel-magic-based rotation?

Or can you tell me how an hevc file is being rotated when rotation is set in lossless cut?

Thank you

**Comment by mifi** [comment]
hi. can you try with a different video player like VLC. does it show the video rotated? if so, then probably a problem with the LG software

**Comment by danergo** [comment]
Everything shows it in correct rotation. Only lg is not.

**Reply by mifi** [reply]
Probably the Lg tv doesn’t support the rotational metadata. Maybe you can try a different format like mkv or mov or mp4
